### DISCUSSION TITLE
choose single phone number from a contact

### DISCUSSION THREAD

**Discussion by bjlockie** [FossifyOrg/General-Discussion#498] [General]
When I start a conversation it shows my contacts with all their phone numbers and if I add a contact it adds all their phone numbers.
When I try to text landline numbers my provider sends me a text saying it is a landline number.
Is there a way to select which number(s) I want to add?
Is there also a way to see what number(s) have been added for a contact?

**Comment by Aga-C** [comment]
When you are creating the conversation, it takes the default number for the contact. Have you set the landline number as a default one?

**Reply by bjlockie** [reply]
Oh, it's not even doing both numbers like I thought. :-(
Why does it show all 3 numbers for a contact when I start a conversation, wouldn't it make sense just to show the default number?
I still want to select the number, I want the default to be landline but mobile for other people.

**Reply by Aga-C** [reply]
There's already a request to add the choice: https://github.com/FossifyOrg/Messages/issues/153. For now, the only way is to temporarily unset the default number - the app will prompt with which number conversation should be created.
